src/coreclr/jit/gentree.cpp

```diff
@@ -17864,6 +17864,57 @@ bool Compiler::gtIsStaticGCBaseHelperCall(GenTree* tree)
     return false;
 }
 
+//------------------------------------------------------------------------
+// gtCallGetDefinedRetBufLclAddr:
+//   Get the tree corresponding to the address of the retbuf taht this call defines.
```

Comment:
```suggestion
//   Get the tree corresponding to the address of the retbuf that this call defines.
```

Reply:
Let me include it as part of #68460 unless there are other changes.